packages/trader/src/Modules/CFD/Components/cfd-account-card.jsx

```diff
@@ -505,11 +504,7 @@ const CFDAccountCard = ({
                 <React.Fragment>
                     {should_show_trade_servers && (
                         <MobileWrapper>
-                            <AddTradeServerButton
-                                ref={button_ref}
-                                onSelectAccount={onSelectAccount}
-                                is_disabled={has_cfd_account_error}
-                            />
+                            <AddTradeServerButton ref={button_ref} onSelectAccount={onSelectAccount} />
```

Comment:
(question) why the logic for `is_disabled` prop is removed?

Reply:
> (question) why the logic for `is_disabled` prop is removed?

IMO, it is crazy too! but if we keep that the user never sees that error I added it into server form modal (https://www.figma.com/file/DYNinH9tDxcqVB0nJ10hDX/DMT5-(Classic-UI))
We correctly made it hidden before but now we have to show it to show that error! :exploding_head: